docs/install_guides/installing-openvino-from-archive-windows.md

```diff
@@ -35,78 +36,75 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
 
 .. tab:: Software
 
-  * For C++ developers:
-     * `Microsoft Visual Studio 2019 with MSBuild <http://visualstudio.microsoft.com/downloads/>`_
-     * `CMake 3.14 or higher, 64-bit <https://cmake.org/download/>`_ (optional, only required for building sample applications)
-  * For Python developers: `Python 3.6 - 3.10, 64-bit <https://www.python.org/downloads/windows/>`_
-     * Note that OpenVINO is gradually stopping the support for Python 3.6. Python 3.7 - 3.10 are recommended.
+  * `Microsoft Visual Studio 2019 with MSBuild <http://visualstudio.microsoft.com/downloads/>`_
+  * `CMake 3.14 or higher, 64-bit <https://cmake.org/download/>`_ (optional, only required for building sample applications)
+  * `Python 3.6 - 3.10, 64-bit <https://www.python.org/downloads/windows/>`_
+     * Note that OpenVINO is gradually phasing out support for Python 3.6. Python 3.7 - 3.10 are recommended.
 
   .. note::
-    You can choose to download Community version. Use `Microsoft Visual Studio installation guide <https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019>`_ to walk you through the installation. During installation in the **Workloads** tab, choose **Desktop development with C++**.
+     To install Microsoft Visual Studio 2019, follow the `Microsoft Visual Studio installation guide <https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019>`_. You can choose to download the Community version. During installation in the **Workloads** tab, choose **Desktop development with C++**.
 
   .. note::
     You can either use `cmake<version>.msi` which is the installation wizard or `cmake<version>.zip` where you have to go into the `bin` folder and then manually add the path to environmental variables.
   
   .. important::
-    As part of this installation, make sure you click the option **Add Python 3.x to PATH** to `add Python <https://docs.python.org/3/using/windows.html#installation-steps>`_ to your `PATH` environment variable.
+    When installing Python, make sure you click the option **Add Python 3.x to PATH** to `add Python <https://docs.python.org/3/using/windows.html#installation-steps>`_ to your `PATH` environment variable.
 
 @endsphinxdirective
 
 ## Installing OpenVINO Runtime
 
 ### <a name="install-openvino"></a>Step 1: Download and Install OpenVINO Core Components
 
```

Comment:
I don't think using command-line to create a new folder, download and extract the package is a user-friendly way on Windows. Taking the user as new and non tech-savvy, I think we should recommend the most common way on Windows. Command-line could be an alternative option.